### PR TITLE
docs(history): add documentation on some management commands

### LINKED ISSUE #873: Document management command for activating history module
There's a management command which needs to be run to "initialise" history when `VersionMixin` is introduced to models with existing data.

The [docs](https://acdh-oeaw.github.io/apis-core-rdf/history.html) currently don't make mention of that.

### PR DESCRIPTION
**Pull request by sennierer**
resolves #873

### CODE REVIEW

File: docs/source/features.rst [hunk 1]
```diff
@@ -48,6 +48,32 @@ can be accessed under `/apis/swagger/schema/swagger-ui/#/apis/apis_api_history_e
 .. _django-simple-history: https://django-simple-history.readthedocs.io/en/latest/
 
 
+Management Commands
+^^^^^^^^^^^^^^^^^^^
+
+The APIS history plugin is based on the `django-simple-history` package. It comes with 
+management commands provided by `django-simple-history`. Some of the more useful commands are:
```
Comment (koeaw) [inline docs/source/features.rst:55]: The first sentence repeats what's already been said at the top of the page. More helpful here would be a direct link to django-simple-history's management commands (docs) for further exploration.

Ex.
The APIS history plugin reuses `django-simple-history`'s [management commands]. Some of the more useful ones are:

File: docs/source/features.rst [hunk 2]
```diff
@@ -48,6 +48,32 @@ can be accessed under `/apis/swagger/schema/swagger-ui/#/apis/apis_api_history_e
...
+  python manage.py populate_history --auto
+
+This command will populate the history tables with the current state of the models. This is useful 
+if you have added the `VersionMixin` to an existing model.
+
+.. code-block:: bash
+
+  python manage.py clean_duplicate_history --auto
+
+This command will clean up duplicate history entries.
```
Comment (koeaw) [inline docs/source/features.rst:68]: Could there be a 1-sentence or half sentence explanation about duplicates added?

Why are there duplicates/how do they come about/why do they need cleaning? Or alternatively a link to where the django-simple-history docs talk about them.

File: docs/source/features.rst [hunk 3]
```diff
@@ -48,6 +48,32 @@ can be accessed under `/apis/swagger/schema/swagger-ui/#/apis/apis_api_history_e
...
+
+.. code-block:: bash
+
+  python manage.py clean_old_history --days 60 --auto
+
+This command will clean up history entries older than 60 days.
+
+
```
Comment (koeaw) [inline docs/source/features.rst:76]: All of the commands mentioned feature the argument `--auto`, but its purpose is never explained.

Would add a note either to the description of the first command (first appearance) or here at the end – or leave if off altogether (if possible/sensible).

File: docs/source/features.rst (hunk 1 above)
Comment (b1rger) [reply docs/source/features.rst:55]: I would not use the term `reuse` for the management commands. The history app does not use those commands, they simply exist, and that only if the `simple_history` app is enabled.

File: docs/source/features.rst (hunk 2 above)
Comment (sennierer) [reply docs/source/features.rst:68]: I dont think that there is an answer to this. Depending on your code you might call save on an object twice (eg via signals) and get duplicates. This management command just checks whether there are exactly the same objects in your history tables and removed one.

**Comment by sennierer** [comment]
Ran the whole part of the documentation through an LLM to improve writing and structure.

**Pull request merged**